docs/dev/table/common.md

```diff
@@ -51,18 +51,18 @@ All Table API and SQL programs for batch and streaming follow the same pattern.
 // create a TableEnvironment for specific planner batch or streaming
 TableEnvironment tableEnv = ...; // see "Create a TableEnvironment" section
 
-// create a Table
-tableEnv.connect(...).createTemporaryTable("table1");
+// create an input Table
+tableEnv.executeSql("CREATE TEMPORARY TABLE table1 ... with ( 'connector' = ... )");
 // register an output Table
-tableEnv.connect(...).createTemporaryTable("outputTable");
+tableEnv.executeSql("CREATE TEMPORARY TABLE outputTable ... with ( 'connector' = ... )");
```

Comment:
nit: could you use the upper case `WITH` instead of `with`?

Reply:
Yes, good point, thanks for the feedback. 
Done.